test/vaadin-combo-box-light.html

```diff
@@ -174,6 +174,50 @@
 
         expect(spy.called).to.be.true;
       });
+
+      describe('custom clear-button', () => {
```

Comment:
BTW Not sure if that feature is useful now:

- With native inputs, there are native clear buttons
- If `<vaadin-text-field>` is used inside `<vaadin-combo-box-light>`, the clear button from the text field could be reused

Reply:
Now that we have a major version bump after this, would it make sense to drop this feature too, perhaps?

Reply:
> With native inputs, there are native clear buttons

Really? 😲

> If <vaadin-text-field> is used inside <vaadin-combo-box-light>, the clear button from the text field could be reused

Good point, but I think if you need `vaadin-combo-box-light.html` it's also likely that you want to use some other input than `<vaadin-text-field>`. This could be supported too though.

I don't really like our current implementation of supporting a custom clear button much, but I think it's still a good feature. I wouldn't probably like to drop the feature completely, but it might make sense to try to rethink how this use case is supported (if we can simplify our code by doing so). Maybe by providing a bit different API to be able to do it with `<vaadin-combo-box-light>` even if it would require a bit more code from the developer using `<vaadin-combo-box-light>` as long as it's not too difficult to use. But I think that would be a separate enhancement.

We could discuss this tomorrow.

Reply:
> Really? 😲

I mean `<input type="search">` 🤓 

Fine by me to keep the feature.